ERCS/erc-7661.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 7661
+title: Extended Multiphase Fractional NFTs
+description: Allows for ERC-721 tokens to be broken into multiple phases of fungible tokens
+author: Jonathan Else (@JEflyer)
+discussions-to: https://ethereum-magicians.org/t/erc-7661-xmf-nfts-extended-multiphase-fractional-nfts/19346
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-03-25
+requires: 165, 721
+---
+
+## Abstract
+
+An extension of the [ERC-721 standard](./eip-721.md) to enable [ERC-721](./eip-721.md) tokens to have multiple phases of fractionalisation into [ERC-20](./eip-20.md) like tokens.
+
+This standard introduces a framework allowing [ERC-721](./eip-721.md) Non-Fungible Tokens (NFTs) to be broken down into phase-based tokens (PB Tokens) representing fractional ownership or states at various lifecycle phases. Unlike standard [ERC-20](./eip-20.md) tokens, PB Tokens  accommodate the multifaceted nature of asset transformation, encapsulating phase transitions that reflect changes in the underlying asset's state, form, or value.
```

Comment:
If you read the contracts you can see the implementation. https://github.com/JEflyer/ERC9988/blob/main/contracts/ERC9988.sol

They are a new type of token, because of the multiphase aspect of the standard they can't be ERC20 because they don't adhere to the interface requirements of ERC20.

This is my first standard I've made, what would you recommend that I put for this standard?

Reply:
I'm only lightly familiar with tokens, and am here to provide mostly editorial feedback :sweat_smile: Maybe you could include some examples of what a "phase" might be, and explain in what scenarios your standard is more useful than say [ERC-7651](https://eips.ethereum.org/EIPS/eip-7651) or [ERC-4675](https://eips.ethereum.org/EIPS/eip-4675).

To be clear, these are suggestions for content to be added to the document itself.